src/inline.d

```diff
@@ -82,9 +82,9 @@ public:
 
     override void visit(Statement s)
     {
-        //printf("Statement::inlineCost = %d\n", COST_MAX);
```

Comment:
It's actually been kind of nice being able to grep for `Class::`.  Though I guess `visit(Class` will work now.